lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -264,6 +269,14 @@ def get_subscriber_data(self, subscriber_id):
                 row = res.fetchone()
                 if not row:
                     raise SubscriberNotFoundError(subscriber_id)
+        except sqlite3.OperationalError:
+            # Print the process holding the lock
+            db_parts = db_location.split(":", 1)
+            if (len(db_parts) == 2) and db_parts[1]:
+                path_str = db_parts[1].split("?")
+                import os
```

Comment:
I don't think there's much benefits into using conditional imports here, unless we have any explicit evidence that there could be performance impact, let's keep this import on the top of the file

Reply:
Comments addressed.